core/templates/dev/head/pages/exploration_player/tutor_card_directive.html

```diff
@@ -266,6 +267,7 @@
       background-color: #0D48A1;
       color: #ffffff;
       padding: 6px 12px;
+      white-space:normal;
```

Comment:
nit: add space after colon

Reply:
Hi @tjiang11 , I will correct it . Thanks for reviewing .